packages/cli-kit/src/public/node/crypto.ts

```diff
@@ -0,0 +1,43 @@
+import crypto from 'crypto'
```

Comment:
Most of these methods could be moved to a `private` module, as they're only in use within CLI kit.

The smaller the surface area, the less we are committing to support.

Reply:
So do you suggest to have this?
- public/node/crypto: randomUUID, hashString
- private/node/crypto: randomHex, randomBytes, base64URLEncode, sha256

While I agree on making public as little as possible, all of the functions here are very similar, so I don't see the point of separating them (why should sha1 be public, but sha256 private?). And I think we won't have to support much because it's very simple and unlikely to change.

Do you still think it's worth separating them?